### ISSUE TITLE
NegativeLogLikelihoodLoss along with NegativeBinomialDistribution not able to model highly seasonal time series

### ISSUE THREAD

**Issue by saurabhtaneja15** [awslabs/gluon-ts#2828]
I have a number of highly seasonal time series data, daily sales of products that sell in high quantity during Thanksgiving and Christmas but very low volume rest of the year (see first figure below). I want to create probabilistic forecasts for such products and I am using NegativeBinomialDistribution along with NegativeLogLikelihoodLoss in DeepAR. Unfortunately DeepAR is not able to capture the peaks during holiday season (see second figure below) and is generating a median forecast output that is way too low for the high season. I feel this is due to the Loss function, as NegativeLogLikelihoodLoss is not penalizing the network for not being able to predict the peaks based on NegativeBinomialDistribution. Is there another distribution or loss function I can use here to model highly right skewed data? Do we need a new loss function to train the model to get the median prediction closer to the peaks? Thanks

![image](https://user-images.githubusercontent.com/153581/235980143-b2787cba-9bde-4054-abad-95c8649a8326.png)

![image](https://user-images.githubusercontent.com/153581/235980342-ddb47b46-b413-4cbb-a729-fe19d5c0b821.png)

**Comment by saurabhtaneja15** [comment]
here is another example - same issue of median forecasts during high season are too low . In this example, there are two consecutive years of high peaks in sales history

![image](https://user-images.githubusercontent.com/153581/235982305-845c1181-33df-4cd6-af8f-a0240971294e.png)

![image](https://user-images.githubusercontent.com/153581/235982422-b0adb202-2409-49bb-b1e8-13701687e2ae.png)

**Comment by saurabhtaneja15** [comment]
I have used the default time_series_features for daily frequency and used Black_Friday, Cyber_Monday and Christmas holidays as dynamic_feat_real.

**Comment by saurabhtaneja15** [comment]
here is another example - this kind of time series can be easily predicted with exponential smoothing or ARIMA but DeepAR is doing a terrible job during high season 

![image](https://user-images.githubusercontent.com/153581/236019779-48b8f16c-9229-4309-b986-54d7c39ab57c.png)
